vms/organization/tests/test_services.py

```diff
@@ -110,12 +113,9 @@ def setup_test_data(cls):
 
         cls.o1.save()
         cls.o2.save()
-        city_name = 'Bothell'
-        state_name = 'Washington'
-        country_name = 'United States'
-        country = get_country_by_name(country_name)
-        state = get_state_by_name(state_name)
-        city = get_city_by_name(city_name)
+        country = create_second_country()
```

Comment:
Why did you change this?

Reply:
There was no country in the `test_vms` database when I was running the test on this app only.
So to create an entry in the database.

Reply:
Did you check it locally or CI?

Reply:
I checked it both ways.
Actually the issue is when we check apps individually, I needed to use `create_` whenever not found exception was there.
But if we run all the apps together, they have common `test_vms` database. So we need to create it only once in any app. And we'll be able to call it in other apps too.

Reply:
Makes sense, then create new Country etc in setUp specifically for these tests and use them only here and in no other app

Reply:
@Monal5031  Sorry I didn't get you exactly.

The latest build for this PR is failing because I called `create_` in volunteer but as This build runs two apps, country etc. were created in prev (organisation app).
How should I manage this? Should I call `get_by` before creating country etc. ?